src/main/product-page-src/README.md

```diff
@@ -0,0 +1,47 @@
+[Documentation](#documentation) |
+[Development](#development) |
+[License](#license)
+
+# [BlueBrainNexus.io](https://bluebrainnexus.io/)
+
+A landing page to get an overview of the technology behind Nexus.
+
+## Development
+
+Anyone can edit this page without needing node or js configured, all you need is docker.
+You want to change some text or add an image? just clone me and make your changes using the steps below!
+
+### With Docker
+
+Make sure you have already installed both [Docker Engine](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/).
+
+- Install: `make install` (installs necessary packages)
```

Comment:
`make` flavour is GNU Make right? Better write it down, since BSD Make can work differently

Reply:
I've made a little note, but I don't really understand it!